app.js

```diff
@@ -0,0 +1,186 @@
+  const popDesc = document.createElement('ul');
+  popDesc.className = 'popDesc';
+  const items1 = ['CANOPY', '• Back End Dev', '• 2015'];
+
+  for (let i = 0; i < items1.length; i += 1) {
+    const li = document.createElement('li');
+    li.innerText = items1[i];
+    popDesc.appendChild(li);
+  }
+
+  // pop iMages
+  const image = document.createElement('img');
+  image.src = projects[a].image;
+  image.className = 'popImg';
+
+  // PopDetails
+  const popDetails = document.createElement('div');
+  popDetails.className = 'popDetails';
+
+  // pop test description
+  const detail = document.createElement('p');
+  detail.innerText = ' Lorem ipsum dolor sit amet consectetur adipisicing elit. Quas molestias iusto architecto blanditiis evenietimpedit tenetur dolorem, non voluptates atque itaque quaerat praesentium incidunt eligendi nostrum animi harumid commodi.Lorem ipsum dolor sit amet consectetur adipisicing elit. Praesentium corporis blanditiis similique neque illomaiores vel repudiandae odit? Facere error suscipit laborum a deleniti rerum dicta quas est corrupti';
+  detail.className = 'popDetails';
+
+  const lang = document.createElement('div');
+  lang.className = 'lang';
+
+  const langTags = document.createElement('ul');
+  langTags.className = 'langTags';
+  const items3 = projects[a].technologies;
+
+  for (let i = 0; i < items3.length; i += 1) {
+    const li = document.createElement('li');
+    li.innerText = items3[i];
+    langTags.appendChild(li);
+  }
+
+  const popBtns = document.createElement('div');
+  popBtns.className = 'popBtns';
+  const items4 = ['See Live', 'See Source'];
+  const images = ['/image/Icon11.png', '/image/Vector11.svg'];
+
+  for (let i = 0; i < items4.length; i += 1) {
+    const li = document.createElement('button');
+    li.className = 'seeBtn';
+    li.innerText = items4[i];
+    const img = document.createElement('img');
+    img.setAttribute('src', images[i]);
+    img.setAttribute('alt', 'icon');
+    li.appendChild(img);
+    popBtns.appendChild(li);
+  }
+
+  // sub divs under lang
+  popDetails.appendChild(detail);
+  popDetails.appendChild(lang);
+
+  lang.appendChild(langTags);
+  lang.appendChild(popBtns);
+
+  popUps.appendChild(popContent);
+  popContent.appendChild(popHeading);
+  popContent.appendChild(popDesc);
+  popContent.appendChild(image);
+  popContent.appendChild(popDetails);
+
+  document.body.appendChild(popUps);
+}
+
+function closePopUp() {
+  const pop = document.getElementById('popUps');
+  pop.style.display = 'none';
+}
+
+// --------------- form validation section ----------------
+
+// eslint-disable-next-line consistent-return
+function validateForm() {
+  const form = document.forms.form1.mail.value;
+  const err = document.getElementsByClassName('error');
+
+  if (form !== form.toLowerCase()) {
+    document.getElementById('error').innerHTML += "Email can't contain capital letters";
+    return false;
+  }
+}
+

```

Comment:
- [ ] Kindly solve the repeated validation message when the email contains an Uppercase latter

![image](https://user-images.githubusercontent.com/65068771/189490081-b71332cf-6dfe-410f-9080-ef1052aef048.png)

Reply:
well resolved! ✔️